bin/start.sh

```diff
@@ -27,13 +27,18 @@ if [ -e /opt/mica/bin/first_run.sh ]
     mv /opt/mica/bin/first_run.sh /opt/mica/bin/first_run.sh.done
 fi
 
-# Wait for MongoDB to be ready
-if [ -n "$MONGO_HOST" ]
-	then
-	until curl -i http://$MONGO_HOST:$MONGO_PORT/mica &> /dev/null
-	do
-  		sleep 1
-	done
+if [ -z ${MONGODB_URI+x} ]
```

Comment:
what's that `+x` ?

Reply:
https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash